test/tools/libtesteth/Options.h

```diff
@@ -53,7 +53,7 @@ class Options
 	std::string rCurrentTestSuite; ///< Remember test suite before boost overwrite (for random tests)
 	bool statediff = false;///< Fill full post state in General tests
 	bool fulloutput = false;///< Replace large output to just it's length
-	bool createRandomTest = false; ///< Generate random test
+	boost::optional<bool> createRandomTest; ///< Generate random test
```

Comment:
What is better with this optional?

Reply:
dunno. Pawel or you suggested to use boost::optional